config/mainnet.go

```diff
@@ -108,7 +108,8 @@ func MainnetConfig() Config {
 			PhaseShift:        240 * time.Hour,
 			CycleGap:          12 * time.Hour,
 			GracePeriod:       1 * time.Hour,
-			RequestRetryDelay: 10 * time.Second,
+			RequestTimeout:    550 * time.Second, // RequestRetryDelay * 2 * MaxRequestRetries*(MaxRequestRetries+1)/2
+			RequestRetryDelay: 5 * time.Second,
```

Comment:
Why change the delay on mainnet?

Reply:
with the given number of max retries it can take up to 18+ Minutes to before the node gives up on fetching from PoET:

10 * 2 * (10*11)/2 = 1100 seconds or 18 Minutes 20 seconds

That felt like overkill, instead of reducing the amount of retries I opted for reducing the delay between the retries. Wdyt?

Reply:
Yeah, it was somewhere between 550 and 1100s. It seems a lot but on the other hand, it also gives us _some_ time to fix poets in case of some hard failure in critical moment.

Reply:
OK I'll increase it to 1100 minutes 🙂